BankingApp/src/com/company/entities/Conturi.java

```diff
@@ -0,0 +1,120 @@
+package com.company.entities;
+
+import java.util.Scanner;
+
+//o clasa mai de sine statatoare pentru main, cu metode in ea, urmeaza de impartit in entities si service
+
+public class Conturi {
+    int sold;
+    int tranzactiePreced;
+    String numeClient;
+    String idClient;
+
+    public Conturi(String cname, String cid) {
+        numeClient = cname;
+        idClient = cid;
+    }
+
+    //e publica pentru ca o folosesc in main la alimentare initiala
+    public void depoziteaza(int suma) {
+        if (suma != 0) {
+            sold = sold + suma;
+            tranzactiePreced = suma;
+        }
+    }
+
+    void retrage(int suma) {
+        if (suma != 0) {
+            sold = sold - suma;
+            tranzactiePreced = -suma;
+        }
+    }
+
+    void gettranzactiePreced() {
+        if (tranzactiePreced > 0) {
+            System.out.println("Tranzactia precedenta, s-a depozitat: " + tranzactiePreced);
+        } else if (tranzactiePreced < 0) {
+            System.out.println("Tranzactia precedenta, s-a retras: " + Math.abs(tranzactiePreced));
+        } else {
+            System.out.println("Nu s-a efectuat tranzactie anterior");
+        }
+    }
+
+    void calculeazaDobanda(int ani) {
+        double rataDobanda = 0.055;
+        double nouSold = (sold * rataDobanda * ani) + sold;
+        System.out.println("Dobanda curenta este: " + (100 * rataDobanda) + "%");
+        System.out.println("Dupa " + ani + " ani, soldul va fi: " + nouSold);
+    }
+
+    public void arataMeniu() {
+        char optiune;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Bun venit, " + numeClient + "!");
+        System.out.println("ID-ul dvs este: " + idClient);
+        System.out.println();
+        System.out.println("Ce ati dori sa faceti?");
+        System.out.println();
+        System.out.println("1. Verificati sold");
+        System.out.println("2. Efectuati o depunere");
+        System.out.println("3. Efectuati o retragere");
+        System.out.println("4. Vedeti tranzactie anterioara");
+        System.out.println("5. Calculati dobanda");
+        System.out.println("6. Iesire");
+
+        do {
+            System.out.println();
+            System.out.println("Alege o optiune: ");
+            char optiune1 = scanner.next().charAt(0);
+            optiune = Character.toUpperCase(optiune1);
+            System.out.println();
+
+            switch(optiune) {
+                //Case '1' se verifica soldul
+                case '1':
+                    System.out.println("====================================");
```

Comment:
[Improvement] logica pentru fiecare case se poate extrage in metode private, e mai curat si mai usor de citit codul asa (eg: handleDeposit(), handleWithdrawal(), etc)